static/app/views/settings/projectMetrics/metricsExtractionRuleForm.tsx

```diff
@@ -526,6 +526,7 @@ export function MetricsExtractionRuleForm({
                               placeholder={t('Add span attributes')}
                               organization={organization}
                               supportedTags={supportedTags}
+                              excludedTags={[]}
```

Comment:
do we need to inform an empty array here?

Reply:
Yes, to override the default from the component. E.g. component filters out "environment" tag by default, and we don't want that behaviour